e2e-tests/package.json

```diff
@@ -28,6 +28,7 @@
     "typescript": "5.5.4"
   },
   "dependencies": {
+    "axios": "^1.7.7",
```

Comment:
It's better to use the built-in functions from @playwright/test for API calls instead of a third-party library. One key reason is that it provides better logging in the HTML reporter and Trace View, making debugging easier.

Reply:
agree! fixed!